packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.ClientModel/src/Providers/RestClientProvider.cs

```diff
@@ -709,5 +721,43 @@ int IEqualityComparer<List<int>>.GetHashCode(List<int> obj)
                 return hash.ToHashCode();
             }
         }
+
+        private static bool TryGetAcceptHeaderContentTypes(
+            InputParameter inputParameter,
+            InputOperation inputOperation,
+            [NotNullWhen(true)] out IReadOnlyList<string>? values)
+        {
+            values = null;
+            if (!inputParameter.NameInRequest.Equals(AcceptHeader, StringComparison.OrdinalIgnoreCase))
```

Comment:
nit: can just use `==` as it uses Ordinal comparison. I don't know that we need to ignore case - we don't ignore case in the other Accept header comparison.